GliaWidgets/CallVisualizer/CallVisualizer.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+import UIKit
+import SalemoveSDK
+
+/// Call Visualizer is a feature for operators to transfer their visitors from conventional comunication methods,
+/// such as a phone call, to Glia solution. The visitor needs to provide to the operator a visitor code,
+/// which they will find in their app either via alert or embedded view. By telling the code to the operator, they can
+/// initiate engagement with the visitor directly through the app.
+///
+/// Call Visualizer class facilitates everything related to this feature:
+///  1. Requesting and displaying visitor code
+///  2. Starting an engagement once the visitor code has been submitted by the operator
+///  3. Handling engagement, featuring video calling, screen sharing, and much more in future.
+public final class CallVisualizer {
+    private var environment: Environment
+    private var visitorCodeCoordinator: VisitorCodeCoordinator?
+
+    init(environment: Environment) {
+        self.environment = environment
+    }
+
+    /// Show VisitorCode for current Visitor
```

Comment:
This line can finish with a period.

Reply:
fixed [11004ba](https://github.com/salemove/ios-sdk-widgets/pull/448/commits/11004ba84b44f7d0cbc82e8cb7c8840bfa9c5246)